routes.yaml

```diff
@@ -0,0 +1,5 @@
+app_name: transactions.web.ch.gov.uk
+group: web
+weight: 910
+routes:
+   1: /transaction/(.*)/confirmation
```

Comment:
I might be on the wrong end of the stick here, is this supposed to be `transaction`? I'm asking because I'm used to seeing `https://api.companieshouse.gov.uk/*transactions*/{transaction_id}/company-accounts/{company_accounts_id}/small-full/approval` alot. If, however the URI is something completely different then ignore me.

Reply:
In the web we use `transaction` - I'm guessing to differentiate between front and back end

Reply:
:+1: